docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -2,12 +2,7 @@
 [role="xpack"]
 == Manage detection alerts
 
-The Alerts page displays all <<detection-alert-def, detection alerts>>.
-From the Alerts table, you can filter alerts, change an alert's status, and start
-investigating and analyzing alerts in Timeline.
-
-TIP: From *Timelines*, you can <<cases-ui-open, create cases>> to track issues and
-share information with colleagues.
```

Comment:
Removed this tip: this topic is about the Alerts page, not Timelines, and the user can create cases directly from the Alerts page; no need to go over to Timelines for that.

Reply:
Good call. I'd also expect for users to attach a Timeline to a case if they were on the Timeline page, not an individual alert.